### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AKostiv8**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,8 +1,7 @@
 Package: shinyScenarioEditor
 Title: Shiny App to edit {esqLABSR} Edit Simulation Scenarios
 Version: 0.0.0.9000
-Authors@R: 
-    person("Felix", "MIL", , "felix.mil@esqlabs.com", role = c("cre", "aut"))
+Authors@R: person('Felix', 'MIL', email = 'felix.mil@esqlabs.com', role = c('cre', 'aut'))
```
Comment (PavelBal) [inline DESCRIPTION:4]: @AKostiv8 Please add yourself to the list of authors.

File: R/app_ui.R [hunk 2]
```diff
@@ -34,6 +35,7 @@ golem_add_external_resources <- function() {
 
   tags$head(
     favicon(),
+    # tags$link(rel = "stylesheet", type = "text/css", href = "style.css"),
```
Comment (PavelBal) [inline R/app_ui.R:38]: Make sure that no commented code is left in the PR.

File: R/mod_edit_table.R [hunk 3]
```diff
@@ -10,34 +10,117 @@
 mod_edit_table_ui <- function(id) {
   ns <- NS(id)
   tagList(
-    datamods::edit_data_ui(ns("edit_df"))
+  # datamods::edit_data_ui(ns("edit_df"))
```
Comment (PavelBal) [inline R/mod_edit_table.R:13]: Commented code

File: R/mod_edit_table.R [hunk 4]
```diff
@@ -10,34 +10,117 @@
...
-      id = "edit_df",
-      download_excel = FALSE,
-      download_csv = FALSE,
-      data_r = reactive(r$data[[tab_section]][[sheet]]$original),
-      reactable_options = list(
-        searchable = TRUE,
-        pagination = FALSE
+    output$edit_df <- renderUI({
+
+      # browser()
```
Comment (PavelBal) [inline R/mod_edit_table.R:27]: ;)

**Review by PavelBal — CHANGES_REQUESTED** [review]
The PR should not contain any commented code. If there are parts you are still working on, then either finish them before merging, or remove and work on them in a separate branch/PR.

**Review by Felixmil — CHANGES_REQUESTED** [review]
- [x] Remove commented code
- [x] Add entries in NEWS describing what is implemented

For next PRs: keep it related to one main change or one topic related implementations.

General recommendations: https://dev.open-systems-pharmacology.org/r-development-resources/collaboration_guide#rules

**Comment by Felixmil** [comment]
Please, add `Remotes` entries in the `DESCRIPTION` file for `{esqlabs.handsontable}` and `{esqlabs.ui}`

File: R/mod_import.R [hunk 5]
```diff
@@ -103,6 +103,11 @@ mod_import_server <- function(id, r, DROPDOWNS) {
       req(projectConfiguration())
       projectConfiguration()$projectConfigurationFilePath
     })
+
+
+    # Share project configuration path with the export module
+    return(projectConfiguration)
```
[the remark this answers is not in the document]
Comment (Felixmil) [reply R/mod_import.R:109]: Also, this value could easily be added to the reactive list `r` in the observer at line 56:
```r
observeEvent(projectConfiguration(), {

   r$configurationPath <- projectConfiguration()$projectConfigurationFilePath

   for (config_file in names(r$data)) {
   ...

File: NEWS.md [hunk 6]
```diff
@@ -1,3 +1,26 @@
-# shinyScenarioEditor (development version)
+#  News
 
-* Initial CRAN submission.
+## Version 0.0.0.1 (2024-05-26)
```
Comment (Felixmil) [inline NEWS.md:3]:
```suggestion
## Version 0.0.0.9000
```

**Review by Felixmil — CHANGES_REQUESTED** [review]

**Review by Felixmil — CHANGES_REQUESTED** [review]

File: README.Rmd [hunk 7]
```diff
@@ -20,70 +24,73 @@ knitr::opts_chunk$set(
...
-# install.packages("devtools")
-devtools::install_github("esqLABS/shinyScenarioEditor")
+```r
+remotes::install_github("esqLABS/shinyScenarioEditor")
 ```
 
-## Example
-
-This is a basic example which shows you how to solve a common problem:
+Note: For projects created for version 3 of `esqlabsR` package, install [`esqlabsRLegacy`](https://github.com/esqLABS/esqlabsRLegacy).
```
Comment (Felixmil) [inline README.Rmd:56]: this is not applicable here

**Review by Felixmil — APPROVED** [review]
Just one line to remove then ok to merge

**Review by PavelBal — APPROVED** [review]

**Pull request merged**